cos-fleet-catalog-tools/camel-connector-maven-plugin/src/main/java/org/bf2/cos/catalog/camel/maven/connector/GenerateCatalogMojo.java

```diff
@@ -544,6 +537,8 @@ private ConnectorDefinition generateDefinitions(KameletsCatalog kamelets, Connec
 
             ObjectNode definition = JSON_MAPPER.convertValue(def, ObjectNode.class);
 
+            importProcessorSchema(definition);
```

Comment:
Can we add a configuration option to enable/disable the processors ? 
So we can merge the pr but gradually enable processors support

Reply:
It should be possible - so disable this by default in connector parent POM and the individual connector explicitly enable it from their own POM?

Reply:
Also do you have a list in your mind to enable it initially? only jms-amqp that I tested?

Reply:
> It should be possible - so disable this by default in connector parent POM and the individual connector explicitly enable it from their own POM?

correct, disable by default and enabled on specific connector when ready

Reply:
> Also do you have a list in your mind to enable it initially? only jms-amqp that I tested?

I don't. 

I'd rater disable it for all as it would significantly impact the catalog.

It should not have impact on the tests.

Reply:
Done